### DISCUSSION TITLE
Saturday 02/12/22 Meeting Agenda

### DISCUSSION THREAD

**Discussion by technobaboo** [omigroup/omi-chan#22] [OMI-Chan Meetings]
OMI-chan Meeting 3 is happening on Saturday February 12th, 2022 from 20 to 21 UTC
 
We'll be meeting again in the omi-weekly-meeting channel of the [AngellXR/OMI Discord](https://discord.gg/NJtT9grz5E).

## Agenda
- Show and tell (5 minutes per person, explain what you're doing in relation to omi-chan)
- Work on avatars?

**Comment by jeanahelver** [comment]
show and tell:
dromiga model sheet sketch
![unknown](https://user-images.githubusercontent.com/32199119/153727384-77c9b8d8-67aa-4de2-ac3e-5df0ef6c5394.png)

oni-chan pre-visualization model
![unknown](https://user-images.githubusercontent.com/32199119/153727404-5c7c3263-584c-4854-9f99-83141b2aacf9.png)
![unknown](https://user-images.githubusercontent.com/32199119/153727436-127ad026-13ab-4fcb-8aed-062a6e17c410.png)
